[2024-09-10T07:28:24.972+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-09-10T07:28:25.032+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_V3.second_task scheduled__2022-07-06T00:00:00+00:00 [queued]>
[2024-09-10T07:28:25.062+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_V3.second_task scheduled__2022-07-06T00:00:00+00:00 [queued]>
[2024-09-10T07:28:25.076+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 6
[2024-09-10T07:28:25.188+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): second_task> on 2022-07-06 00:00:00+00:00
[2024-09-10T07:28:25.229+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=15778) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2024-09-10T07:28:25.236+0000] {standard_task_runner.py:72} INFO - Started process 15804 to run task
[2024-09-10T07:28:25.251+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_V3', 'second_task', 'scheduled__2022-07-06T00:00:00+00:00', '--job-id', '3483', '--raw', '--subdir', 'DAGS_FOLDER/our_first_dag.py', '--cfg-path', '/tmp/tmpvqo3h2o9']
[2024-09-10T07:28:25.272+0000] {standard_task_runner.py:105} INFO - Job 3483: Subtask second_task
[2024-09-10T07:28:25.455+0000] {task_command.py:467} INFO - Running <TaskInstance: our_first_dag_V3.second_task scheduled__2022-07-06T00:00:00+00:00 [running]> on host 5508ea9ebfaf
[2024-09-10T07:28:25.682+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='coder2j' AIRFLOW_CTX_DAG_ID='our_first_dag_V3' AIRFLOW_CTX_TASK_ID='second_task' AIRFLOW_CTX_EXECUTION_DATE='2022-07-06T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-07-06T00:00:00+00:00'
[2024-09-10T07:28:25.683+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-09-10T07:28:25.715+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-09-10T07:28:25.717+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, this is the second task!']
[2024-09-10T07:28:25.746+0000] {subprocess.py:86} INFO - Output:
[2024-09-10T07:28:25.753+0000] {subprocess.py:93} INFO - hey, this is the second task!
[2024-09-10T07:28:25.761+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-09-10T07:28:25.836+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-09-10T07:28:25.837+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=our_first_dag_V3, task_id=second_task, run_id=scheduled__2022-07-06T00:00:00+00:00, execution_date=20220706T000000, start_date=20240910T072825, end_date=20240910T072825
[2024-09-10T07:28:25.883+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-09-10T07:28:25.911+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-09-10T07:28:25.916+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
